src/data/uma-title.csv

```diff
@@ -32,7 +32,7 @@
 "[tach-nology]","[tach-nology]"
 "[Go To Winning!]","[Go To Winning!]"
 "[フィーユ・エクレール]","[Fille Éclair]"
-"[オーセンティック/1928]",""
+"[オーセンティック/1928]","Authentic/1928"
```

Comment:
you missed the [ and ]

Reply:
Oops. Time for new PR.

Reply:
also for two pieces

Reply:
Two Pieces looks fine on my end.

Reply:
nvm, i think i included it in my pr haha